source/administration/command-line-tools.rst

```diff
@@ -1380,6 +1542,120 @@ mattermost version
     .. code-block:: none
 
       mattermost version
+      
+mattermost webhook
+---------------
+
+  Description
+    Commands to manage webhooks.
+
+  Child Commands
+    -  `mattermost webhook create-incoming`_ - Create incoming webhook within specific channel. 
```

Comment:
I'm curious if `mattermost webhook incoming`  with options to `--create` and `--modify` might be more consistent with our other commands.

Reply:
I can discuss with the team.  I know we wanted to keep these simple for now, so likely this was the easiest implementation.  Are you ok with me resolving this and creating a follow up ticket with the team?

Reply:
Yeah, if the doc follows what we currently have in the code, okay to merge.